internal/proto/encryptionkeyrotation.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.34.2
-// 	protoc        v3.20.2
+// 	protoc        v3.19.6
```

Comment:
We should keep this to the same version as we have in GitHub action https://github.com/csi-addons/kubernetes-csi-addons/blob/bd88536de3da60a31dc293c8654ca9eb11589bca/build/Containerfile.tools#L14 is the place we are installing this in fedora or remove installing protobuf in the GitHub action, we need to be consistency

Reply:
I'd prefer to remove the protoc installation in the github action. The CI job calls `make test`, so it should run in the container now too.

https://github.com/csi-addons/kubernetes-csi-addons/blob/main/.github/workflows/test-golang.yaml#L23-L27

Reply:
Once this is addressed, it should be good to go.